tests/test_pathutils.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from manage.pathutils import Path, PurePath
+
+def test_path_match():
+    p = Path("python3.12.exe")
+    assert p.match("*.exe")
+    assert p.match("python*")
+    assert p.match("python*.exe")
```

Comment:
`*` can be a null match as far as I can tell, add a test case for this
```suggestion
    assert p.match("python*.exe")
    assert p.match("python3.12*.exe")
```